slog/slog.go

```diff
@@ -184,7 +184,7 @@ const (
 	loggerKey key = iota
 )
 
-func validateLogLevel(level string) (Level, error) {
+func ParseLevel(level string) (Level, error) {
```

Comment:
Maybe add some documentation for them since now they are public. Just curious why are they being made public ? The LoadConfig function is not working on some scenario ?

Reply:
Sorry didn't think about it. Added the doc and add also a testcase.

Yeah, I'm reading other envvars on my code already, so I thought I should read everything and just send the config. Just a different approach I guess.